autogen/main/dns.tf.tmpl

```diff
@@ -44,7 +44,7 @@ EOF
   ]
 }
 
-resource "kubernetes_config_map_v1_data" "kube-dns-upstream-namservers" {
+resource "kubernetes_config_map_v1_data" "kube-dns-upstream-nameservers" {
```

Comment:
@apeabody I know a couple other resources in this file are using kebab case as well, but is it worth switching it to snake case at the same time, since it's being adjusted?
Most of the resources in this module use snake case, and it is recommended most of the time (including [here](https://cloud.google.com/docs/terraform/best-practices/general-style-structure#naming-convention).

Reply:
Hey @wyardley! As this is a correction to an existing name I'm fine with either, but yes, for new resources we should try to use snake case.

Reply:
@wyardley, happy to see you! :)

While I generally agree that we should all adhere to the same snake_case convention, I believe it would be better to address this consistently - either within this file or across the entire module. This might even warrant a dedicated `moved.tf` file of its own.
Updating a single variable from _kebab-case_ to _snake_case_ likely isn't worth the effort.

What do you guys think?

Reply:
Let's leave as-is given the current mix in the module for a future PR to update everything.